RecoPixelVertexing/PixelTrackFitting/interface/BrokenLine.h

```diff
@@ -0,0 +1,614 @@
+#ifndef RECOPIXELVERTEXING_PIXELTRACKFITTING_BROKENLINE_H
+#define RECOPIXELVERTEXING_PIXELTRACKFITTING_BROKENLINE_H
+
+#include <Eigen/Core>
+#include <Eigen/Cholesky>
+#include <Eigen/Eigenvalues>
+#include <cuda.h>
+
+#ifdef __CUDACC__
+#define CUDA_HOSTDEV __host__ __device__
```

Comment:
@fwyzard Wasn't it sufficient to just use `__host__ __device__` everywhere? Or did it require `#include <cuda_runtime.h>`?

Reply:
Just copied from RiemannFit.h ...

Reply:
Indeed, please change
```c++
#include <cuda.h>

#ifdef __CUDACC__
#define CUDA_HOSTDEV __host__ __device__
#else
#define CUDA_HOSTDEV
#endif
```
with just
```c++
#include <cuda_runtime.h>
```

Reply:
And all the `CUDA_HOSTDEV` with `__host__ __device__`?